File: oop/BaseAdvertising.java
package oop;

public class BaseAdvertising {
    
    private int ID; 
    private int clicks = 0; 
    private int views = 0; 

    public BaseAdvertising() {

    }

    public int getID() {
        return this.ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public int getClicks() {
        return this.clicks;
    }

    public void incClicks() {
        this.clicks += 1;
    }

    public int getViews() {
        return this.views;
    }

    public void incViews() {
        this.views += 1;
    }

    public String describeMe() {

        return "Class BaseAdvertising";
    }
}

File: python/baseadvertising.py
#!/usr/bin/python3 

class BaseAdvertising():

    _clicks = 0 
    _views = 0 
    
    def __init__(self, id) -> None:
        self._id = id 

    def getId(self):
        return self._id 

    def setId(self, id):
        self._id = id

    def getClicks(self):
        return self._clicks 

    def incClicks(self):
        self._clicks += 1 

    def getViews(self): 
        return self._views 

    def incViews(self):
        self._views += 1

    @staticmethod 
    def describeMe():
        return "Class BaseAdvertising"